presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
```

Comment:
We don't use italics.

Reply:
This should specify the configuration property and session property names. See previous release notes for the standard way that we phrase this.

Reply:
I think it's fine to use them here.